tasks/generate-history/createSarcophagi.ts

```diff
@@ -0,0 +1,170 @@
+import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers";
+import { BigNumber } from "ethers";
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+import { split } from "shamirs-secret-sharing-ts";
+import { generateHistoryConfig } from "./config";
+import { range, signHre } from "./helpers";
+
+export interface SarcophagusData {
+  sarcoId: string;
+  // A mapping of archaeologist address to the encrypted shard
+  shards: { [key: string]: Uint8Array };
+}
+
+/**
+ * Creates some sarcohpagi provided some archaeologists.
+ * @param hre The Hardhat Runtime Environment
+ * @param archaeologistSigners The archaeologist signers
+ * @param resurrectionTime The resurrection time in seconds
+ */
+export async function createSarcophagi(
+  hre: HardhatRuntimeEnvironment,
+  archaeologistSigners: SignerWithAddress[],
+  resurrectionTime: number
+): Promise<SarcophagusData[]> {
+  const maxArchaeologistOnSarcophagus = 25;
+  const sarcophagusCount = generateHistoryConfig.sarcophagusCount;
+
+  // Define constants
+  const fakePayloadTxId = "fake-payload-tx-id";
+  const fakeShardTxId = "fake-shard-payload-id";
+  // Needs to be far enough in the future that the sarcophagus will be created
+  // Get hre ethers tools
+  const { ethers } = hre;
+  const { randomBytes, hexlify, keccak256 } = ethers.utils;
+  const { getSigners } = ethers;
+
+  const timestamp = (await ethers.provider.getBlock("latest")).timestamp;
+  const maximumRewrapInterval = 604800;
+  const diggingFee = 100;
```

Comment:
Left a couple comments on the register archaeologist about randomizing these values, and that would affect these -- the digging fees would need to be set to a per-archaeologist value (based on their `minDiggingFee`) and the `maximumRewrapInterval` may need to be set to the maximum of the `maximumRewrapInterval` values (unless this value is automatically larger than the randomized vals, then it could be static)